attalos/dataset/visgenome-densecap/visgenome2attalos.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
```

Comment:
Did you mean to check this in? It looks like this could be done from the repo that was checked out, not by `wget`ing the repo.